test/es6-import.js

```diff
@@ -0,0 +1,12 @@
+import jsdomify from 'jsdomify';
+import {assert} from 'chai';
+import Cookies from '../src/js.cookie';
+
+it('should function properly when jsdom is used', () => {
```

Comment:
QUnit runs in the browser and node. If we want to run in node we can use nodeunit. The [example I posted in this comment](https://github.com/js-cookie/js-cookie/pull/237#issuecomment-232860139) shows how we are testing in node without having to use Mocha.